### PR TITLE
download release history

### PR DESCRIPTION
**Pull request by GitEdvard**
Release history contains the name, date and description of each release
at a given repo. These fields are fetched from the json object from
GitHub.

Intended workflow: After accepting a release, the user goes to GitHub
and fill in the description manually in release-history. After that the
download-release-history command is given.

### CODE REVIEW

File: release_tools/github.py [hunk 1]
```diff
@@ -88,6 +89,25 @@ def download_archive(self, branch, save_to_path, ball="zipball"):
             archive.extractall(save_to_path)
             print "Extracted"
 
+    def download_release_history(self, path):
+        url = "https://api.github.com/repos/{owner}/{repo}/releases{token}"\
+              .format(owner=self.owner, repo=self.repo, token=self.access_token_postfix())
+        response = requests.get(url)
+        if response.status_code == 200:
+            print "Writing to file..."
```
Comment (withrocks) [inline release_tools/github.py]: It wouldn't hurt if this was changed to the print function (`from __future__ import print_function`) so it's ready for py3. Or change it to `logging.info(...)`?

File: release_tools/github.py [hunk 2]
```diff
@@ -88,6 +89,25 @@ def download_archive(self, branch, save_to_path, ball="zipball"):
...
+            with open(path, 'w') as f:
+                f.write(self._release_history_contents(response.json()))
+            print "done."
+        else:
+            print "Something went wrong, contents cannot be downloaded"
```
Comment (withrocks) [inline release_tools/github.py]: Alternatively one could raise an error here and only write this message in the `cli.py`, then the module is easier to use as a library.

**Review by withrocks — APPROVED** [review]
Looks cool. Two comments, let me know if you want to make changes or if I should merge it as it is.

**Comment by GitEdvard** [comment]
Ok, I've added to commits for the print syntax, and for changing an error print to an exception.

**Review by withrocks — APPROVED** [review]

**Pull request merged**